docs/detections/session-view.asciidoc

```diff
@@ -0,0 +1,78 @@
+[[session-view]]
+== Session view
+This page describes how to enable and use Session View. Session View is an investigation tool that allows you to view and analyze Linux process data organized
+in a tree according to the Linux logical event model, with processes organized by parentage and time of execution.
+It displays events in a highly readable format inspired by the terminal, making it a powerful tool for monitoring
+and investigating remote session activity on your Linux infrastructure, and understanding user and service behaviour.
+It is currently in beta.
+
+[float]
+[[session-view-data]]
+====== Session View displays:
+1. *Interactive and non-interactive processes:* processes and services with or without a controlling tty.
+2. *User information:* the user that executed each session or process, and any exec user changes.
+3. *User Input:* interactive processes entered by a user into the terminal.
+4. *Output:* output generated as a result of process activity.
+5. *Process and event telemetry:* process information included in the Linux logical event model.
+6. *Nested sessions:* sessions started by processes descended from the entry session.
+
+[float]
+[[enable-session-view]]
+=== Enable session view
+Session View uses process data collected by the {endpoint-sec} integration,
+but this data is not collected by default. To enable Session View, navigate to the Endpoint Security
+integration and edit one or more of your integration policies. Search the policy's settings for “session data”,
+and enable “Include session data” near the bottom of the page. Only data collected by agents with this setting
```

Comment:
please change "agents" to "Endpoint" we want to make it clear that Elastic Agent is NOT collecting this data and it's the responsibility of Endpoint